scripts/Refinery.js

```diff
@@ -186,67 +210,70 @@ async function RefinerySwap(sourcetokenAddress, targetTokenAddress, amount) {
-    if (isTargetTokenFoundry === true) {
-      console.log('Target token is foundry asset')
-      console.log('withdraw foundry asset ...')
+    // If it is a foundry token 
+    if (isTargetRefineryAsset === false) {
+      console.log('TN-1 FoundrySwap: Target Token is a Foundry Asset')
+      console.log('TN-2 FoundrySwap: Withdraw FoundryToken from FundManager to User ...')
 
       //if target token is foundry asset
       // Call FiberRouter to withdraw tokens from FundManager to user's account
-      const result = await targetFiberRouterContract.connect(targetSigner).withdraw(
+      const result1 = await targetFiberRouterContract.connect(targetSigner).withdraw(
         targetFoundryTokenAddress, //token address on network 2
         signer.address, //receiver
         amountsOut, //targetToken amount
         { gasLimit: 1000000 }
       );
 
-      const receipt = result.wait();
+      const receipt = result1.wait();
       if (receipt.status == 1) {
-        console.log("success withdraw foundry token on source network !");
+        console.log("TN-3 SUCCESS: Withdraw FoundryToken on Source Network (SN)!");
+        console.log("Transaction hash is: ", result1);
       }
-
     } else {
-      console.log('target token is refinery asset')
-      console.log('withdraw foundry token ....')
+      console.log('TN-1 RefinerySwap: Target Token is RefineryAsset')
 
-      //if target token is not foundry asset
-      console.log('swap foundry token to target token ....')
+      console.log('TN-2 RefinerySwap: Withdraw FoundryToken from FundManager for Swap ....')
 
-      // Swap pair (Foundry, Target)
+      console.log('TN-3 RefinerySwap: Swap FoundryToken to TargetToken from DEX....')
+
+      // Swap pair (Foundry, Target)  --> (usdt, cake)
       let path2 = [targetFoundryTokenAddress, targetTokenAddress];
 
       // Check the swapOut amount of target tokens to be received from DEX
+       // 9.89 usdt --> 8 cake 
       const amounts2 = await targetDexContract.getAmountsOut(amountsOut, path2);
       const amountsOut2 = amounts2[1];
 
       // Call FiberRouter to perform swap on DEX 
       // Transfer tokens from user wallet to DEX Router
-      const result3 = await targetFiberRouterContract.connect(targetSigner).withdrawAndSwap(
+      const result2 = await targetFiberRouterContract.connect(targetSigner).withdrawAndSwap(
         signer.address, // user wallet address
         targetRouterAddress, // DEX Address
-        amountsOut,  // the amount that needs to be swapped to TargetToken
+        amountsOut,  // the amount of foundrytoken that needs to be swapped to TargetToken
         amountsOut2, // the amount that will be received after swap from DEX
         path2, // targetToken pair (Foundry, Target)
         '1669318064', // Timeout
         {
           gasLimit: 1000000,
         }
       );
-      const receipt3 = await result3.wait();
+      const receipt3 = await result2.wait();

```

Comment:
result2 variable name would be good to be updated to something like `swapResult` or something that looks more relevant.